admin/src/Components/Layout/Layout.tsx

```diff
@@ -0,0 +1,152 @@
+          <ListItemText primary="Products" />
+        </ListItem>
+
+        <ListItem button onClick={() => navigate("/references")}>
+          <ListItemText primary="References" />
+        </ListItem>
+
+        <ListItem button onClick={() => navigate("/pages")}>
+          <ListItemText primary="Pages" />
+        </ListItem>
+
+        <ListItem button onClick={() => navigate("/users")}>
+          <ListItemText primary="Users" />
+        </ListItem>
+      </List>
+    </div>
+  );
+
+  const container =
+    window !== undefined ? () => window().document.body : undefined;
+
+  return (
+    <Box sx={{ display: "flex" }}>
+      <CssBaseline />
+      <AppBar
+        position="fixed"
+        sx={{
+          width: { sm: `calc(100% - ${drawerWidth}px)` },
+          ml: { sm: `${drawerWidth}px` },
+        }}
+      >
+        <Toolbar>
+          <IconButton
+            color="inherit"
+            aria-label="open drawer"
+            edge="start"
+            onClick={handleDrawerToggle}
+            sx={{ mr: 2, display: { sm: "none" } }}
+          >
+            <MenuIcon />
+          </IconButton>
+          <Typography variant="h6" noWrap component="div">
+            Responsive drawer
+          </Typography>
+        </Toolbar>
+      </AppBar>
+      <Box
+        component="nav"
+        sx={{ width: { sm: drawerWidth }, flexShrink: { sm: 0 } }}
+        aria-label="mailbox folders"
+      >
+        {/* The implementation can be swapped with js to avoid SEO duplication of links. */}
+        <Drawer
+          container={container}
+          variant="temporary"
+          open={mobileOpen}
+          onClose={handleDrawerToggle}
+          ModalProps={{
+            keepMounted: true, // Better open performance on mobile.
+          }}
+          sx={{
+            display: { xs: "block", sm: "none" },
+            "& .MuiDrawer-paper": {
+              boxSizing: "border-box",
+              width: drawerWidth,
+            },
+          }}
+        >
+          {drawer}
+        </Drawer>
+        <Drawer
+          variant="permanent"
+          sx={{
+            display: { xs: "none", sm: "block" },
+            "& .MuiDrawer-paper": {
+              boxSizing: "border-box",
+              width: drawerWidth,
+            },
+          }}
+          open
+        >
+          {drawer}
+        </Drawer>
+      </Box>
+      <Box
+        component="main"
+        sx={{
+          flexGrow: 1,
+          p: 3,
+          width: { sm: `calc(100% - ${drawerWidth}px)` },
+        }}
+      >
+        <Toolbar />
+
+        <Routes>
+          <Route path="/" element={<div>Orders</div>} />
+          <Route path="/products" element={<div>Products</div>} />
+          <Route path="/references" element={<div>References</div>} />
+          <Route path="/pages" element={<div> Pages </div>} />
+          <Route path="/users" element={<div> Users </div>} />
+        </Routes>
```

Comment:
Try to replace the Routes component here with an Outlet and put the Routes into App.tsx, will make more sense later on when we generalize the Admin App i think.

As an Example:
https://stackblitz.com/edit/github-6zktaw?file=src/App.tsx